app/institutions/dashboard/-components/institutional-users-list/template.hbs

```diff
@@ -1,176 +1,118 @@
-                    @triggerClass={{local-class 'select'}}
-                    @search={{perform this.searchDepartment}}
-                    @onChange={{action this.onSelectChange}}
-                    as |department|
-                >
-                    {{department}}
-                </PowerSelect>
-            </div>
+    <div local-class='filter-container'>
+        <div local-class='orcid-switch'>
+            <label for='switches-container' local-class='orcid-toggle-label'>{{t 'institutions.dashboard.users_list.has_orcid'}}</label>
+            <label local-class='switch'>
+                <input id='switches-container' type='checkbox' checked={{this.hasOrcid}} {{on 'change' (fn this.toggleOrcidFilter (not this.hasOrcid))}}>
+                <span local-class='slider round'></span>
+            </label>
         </div>
+        <PowerSelect
+            @options={{this.departments}}
+            @searchEnabled={{true}}
+            @selected={{this.department}}
+            @triggerClass={{local-class 'select'}}
+            @search={{perform this.searchDepartment}}
+            @onChange={{this.onSelectChange}}
+            as |department|
+        >
+            {{department}}
+        </PowerSelect>
+        <BasicDropdown
+            @renderInPlace={{true}}
+            @onClose={{this.onDropdownClose}}
+            @preventScroll={{true}}
+            @eventType='mousedown'
+            local-class='dropdown'
+            as |dd|
+        >
+            <button type='button' local-class='dropdown-trigger' {{on 'click' dd.actions.toggle}}>
+                <FaIcon local-class='icon-columns' @icon='columns' />
+                {{t 'institutions.dashboard.users_list.select_columns'}}
+            </button>
+            {{#if dd.isOpen}}
+                <div local-class='dropdown-panel'>
+                    <div local-class='dropdown-content'>
+                        {{#each this.columns as |column|}}
+                            <label>
+                                <input
+                                    type='checkbox'
+                                    value={{column.key}}
+                                    checked={{column.value}}
+                                    {{on 'change' (fn this.toggleColumnSelection column.key)}}
+                                />
+                                {{column.label}}
+                            </label>
+                        {{/each}}
+                    </div>
+                    <div local-class='dropdown-actions'>
+                        <button type='button' {{on 'click' dd.actions.close}}>
+                            {{t 'institutions.dashboard.users_list.cancel'}}
+                        </button>
+                        <button type='button' local-class='apply-button' {{on 'click' this.applyColumnSelection}}>
+                            {{t 'institutions.dashboard.users_list.apply'}}
+                        </button>
```

Comment:
Similarly, a lot of the styling can be simplified by turning these into `<Button>`s as well. For the "Apply" button, you'll want to use `<Button @type='primary'>` to turn it into a blue button.